static/js/base.js

```diff
@@ -549,3 +549,8 @@ function log(text) {
     }
     consoleLog.scrollTop = consoleLog.scrollHeight;
 }
+
+let clear_console = () => {
```

Comment:
could make a traditional function i.e. `function clear_console()`

also change to snake case i.e. `clearConsole`